README.md

```diff
@@ -20,20 +32,36 @@ Thanks to the support for K3s, also single machines can participate,creating dyn
 
 Liqo leverages the same highly successful “peering” model of the Internet, without any central point of control. New peering relationships can be established dynamically, whenever needed, even automatically. Cluster auto-discovery can further simplify this process.
 
-## Features
-
-* Dynamic discovery of clusters in LAN
-* Seamless pod execution on remote cluster,
-* Seamless reconciliation on remote clusters of K8s objects (i.e. configmaps, secrets, services, endpoints)
-
 ## Quickstart
 
-Liqo can be installed via Helm.
-The parameters of the home cluster required by Liqo to start should be automatically discovered by the Liqo installer 
-(through proper kubeadm calls).
+Create two [KinD](kind.sigs.k8s.io/) clusters via our script or bring your own clusters. 
+N.B. Using our cluster, Docker has to be installed on your test machine and user should have permission to issue commands.
 
 ```bash
-curl https://raw.githubusercontent.com/liqotech/liqo/master/install.sh | bash
+source <(curl -L https://get.liqo.io/clusters.sh)
+```
+
+Install Liqo on both clusters:
+
+```
+export KUBECONFIG=$KUBECONFIG_1
+curl -L https://get.liqo.io | bash -s
+export KUBECONFIG=$KUBECONFIG_2
+curl -L https://get.liqo.io | bash -s
+```
+
+Wait that all containers are up and running (around 30 seconds). When a new virtual-kubelet pops out, a new node modeling the remote cluster is present
+
+```
+kubectl get no -o wide
+```
+
+Let's use those resources. Deploy the [Google microservice Shop](https://github.com/liqotech/microservices-demo/blob/master/release/kubernetes-manifests.yaml) application via: 
+
+```
+kubectl create ns test-liqo
```

Comment:
can we change the namespace name? I would use "demo", or something similar (I would avoid the "test" word)